packages/navigation/src/hooks/useWebviewNavigate.ts

```diff
@@ -160,10 +163,7 @@ export function useWebviewNavigate<
         : getStateFromPath(path, options?.config);
 
       if (state) {
-        // for REPLACE action we need to pass only top of navigation state.
-        const actionState =
-          actionType === 'replace' ? unpackState(state) : state;
```

Comment:
No need to unpack state as the `getMinimalAction` essentially does the same, but supporting nested structures correctly.

Reply:
`unpackState` import and utility method should be removed in this case